.github/scripts/check-grpc-coverage.py

```
@@ -158,50 +122,54 @@ def check_query_implementation(query_name, sdk_path):
+    check_script = wasm_sdk_path / "check_documentation.py"
 
     if not check_script.exists():
-        return True, []  # Skip if WASM SDK doesn't have the check script
+        return True, []
 
-    # Run the documentation check
     import subprocess
-    result = subprocess.run(['python3', str(check_script)], cwd=wasm_sdk_path, capture_output=True, text=True)
 
-    # Parse the output to find errors
+    result = subprocess.run(
```

Ruff (0.15.4)

[error] `subprocess` call: check for execution of untrusted input

(S603)